scalableminds/webknossos
# Make brush size a user setting

This allows to remember the user's brush size setting across page reloads and to set it using the recommended configuration.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a volume tracing and change the brush size.
- Reload the page - the brush size should be the same.
- Create a volume task with a recommendedConfiguration that contains `brushSize: ...`.
- Open the task and accept the recommended configuration - the brush size should have the defined value.

### Issues:
- fixes #3779 

------
